debian/calaos-server.service

```diff
@@ -1,9 +1,13 @@
 [Unit]
-Description=Calaos-Server Daemon
+Description=Calaos home automation server service
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/calaos-server
+ExecStart=/usr/local/bin/calaos_server
```

Comment:
I don't think /usr/local/bin is the prefix used in a default installation. We should use a calaos-server.service.in and create calaos-server.service at configuration time.